src/components/RightSideDrawer/RightSideDrawer.js

```diff
@@ -0,0 +1,57 @@
+import { Drawer, makeStyles } from "@material-ui/core"
+import AddNewCharacterForm from "../AddNewCharacterForm"
+
+const useStyles = makeStyles((theme) => ({
+  paperAnchorRight: {
+    backgroundColor: "transparent",
+  },
+  root: {
+    display: "flex",
+    fontFamily: theme.typography.fontFamily,
+  },
+  closeContainer: {
+    height: "inherit",
+    color: theme.palette.common.white,
+    fontWeight: theme.typography.fontWeightBold,
+    fontSize: "small",
+    "& div": {
+      cursor: "pointer",
+      padding: "8px 15px 10px 15px",
+      borderRadius: "0 0 0 5px",
+      backgroundColor: "#375a57",
+      height: "3%",
+    },
+  },
+  form: {
+    backgroundColor: theme.palette.common.white,
+    height: "100vh",
+  },
+}))
+
+const RightSideDrawer = ({ open, setOpen }) => {
```

Comment:
What if we need to use this form in another page, what can you do to make this for reusable?